RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  GoogleMapsHelper.swift
+//  RoutesApp_ios
+//
+//  Created by user on 19/9/22.
+//
+
+import Foundation
+import GoogleMaps
+
+class GoogleMapsHelper {
+    static var shared = GoogleMapsHelper()
+
+    func drawPolyline(map: GMSMapView, list: [Coordinate], hexColor: String = "#004696") {
+        let path = GMSMutablePath()
+        list.forEach({ path.add($0.toCLLocationCoordinate2D()) })
+        let polyline = GMSPolyline(path: path)
+        polyline.strokeColor = ColorHelper.shared.hexStringToUIColor(hex: hexColor)
+        polyline.strokeWidth = 5.0
+        polyline.geodesic = true
+        polyline.map = map
+    }
+
+    func drawDotPolyline(map: GMSMapView, path: GMSPath, hexColor: String = "#004696") {
```

Comment:
add this default color as a constant

Reply:
Added the default polyline color as constant